locales/en-US/app.ftl

```diff
@@ -1,21 +1,21 @@
 siteName = Firefox Test Pilot
 
-# Page titles, put in the <title> HTML tag.
+// Page titles, put in the <title> HTML tag.
```

Comment:
The summary of syntax changes is available at https://github.com/l20n/l20n.js/blob/master/CHANGELOG.md.